cmd/melt/main.go

```diff
@@ -67,8 +76,12 @@ be used to rebuild your public and private keys.`,
 
 				// Build formatted restore command
 				const cmdEOL = " \\"
+				var lang string
+				if language != "en" {
+					lang = fmt.Sprintf(" -l %s", language)
```

Comment:
since we use the full flag on `--seed`, maybe we can use the full `--language` here too